http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1186,11 +1186,6 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-      security:
```

Comment:
These were blocking Azure logins for the app routes in Swagger. We don't need a `security` field for each route because it's set globally.